### PR TITLE
Add region parameter

### PR DESCRIPTION
**Pull request by Nerothos**
Add a third parameter to select the API region.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "battle-net-oauth"
-version = "0.1.8"
-authors = ["Shempignon <nicolas@pointndots.com>"]
+version = "0.1.9"
+authors = ["Nerothos"]
```
Comment (shempignon) [inline Cargo.toml]: ?

File: Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 edition = "2018"
 documentation = "https://docs.rs/battle-net-oauth/"
 readme = "README.md"
-repository = "https://github.com/shempignon/battle-net-oauth"
+repository = "https://github.com/nerothos/battle-net-oauth"
```
Comment (shempignon) [inline Cargo.toml]: ?

**Review by shempignon — CHANGES_REQUESTED** [review]
I am overall Ok with the changes you propose, but why did you change the crate author ?

**Comment by Nerothos** [comment]
Ah, sorry, I probably committed the file by mistake. I'm gonna revert it.

Edit: Reverted, sorry again, it was mass replace for testing purpose.

**Review by shempignon — APPROVED** [review]

**Comment by shempignon** [comment]
Thanks @Nerothos

**Pull request merged**